be/src/vec/sink/writer/vtablet_writer.cpp

```diff
@@ -17,6 +17,7 @@
 
 #include "vtablet_writer.h"
 
+#include <bits/ranges_algo.h>
```

Comment:
The header `bits/ranges_algo.h` isn't portable.
```suggestion
#include <ranges>
```

Reply:
yes. will fix immediately